R/create.R

```diff
@@ -684,20 +672,17 @@ create_initial_conditions <- function(data) {
       out$bp_sd <- array(numeric(0))
       out$bp_effects <- array(numeric(0))
     }
-    if (data$obs_scale_sd > 0) {
-      out$frac_obs <- array(truncnorm::rtruncnorm(1,
-        a = 0, b = 1,
-        mean = data$obs_scale_mean,
-        sd = data$obs_scale_sd
-      ))
-    } else {
-      out$frac_obs <- array(numeric(0))
-    }
     if (data$week_effect > 0) {
       out$day_of_week_simplex <- array(
         rep(1 / data$week_effect, data$week_effect)
       )
     }
+    out$params <- array(truncnorm::rtruncnorm(
+      data$n_params_variable,
+      a = data$params_lower,
+      b = data$params_upper,
+      mean = 0, sd = 1
```

Comment:
why is this initialising at 0, 1 and not the mean and sd as before?

Reply:
Because we don't have a way of determining the mean/sd once we've done the conversion to stan code. My suggestion was 

> will have to monitor if this leads to any issues.

but if we're not happy with that we could add a function for going back from data to `<dist_spec>` and then use its functionality to get the mean and sd.

Reply:
Because we don't have a way of determining the mean/sd once we've done the conversion to stan code. My suggestion was 

> will have to monitor if this leads to any issues.

but if we're not happy with that we could add a function for going back from data to `<dist_spec>` and then use its functionality to get the mean and sd.